src/methods/aggregate.jl

```diff
@@ -259,10 +264,10 @@ Disaggregate array `src` to array `dst` by some scale, using `method`.
 Note: currently it is faster to aggregate over memory-backed arrays.
 Use [`read`](@ref) on `src` before use where required.
 """
-function disaggregate!(locus::Locus, dst::AbstractRaster, src, scale)
+function disaggregate!(dst::AbstractRaster, src, scale)
```

Comment:
Don't understand this one - Isn't this the exact same type signature as the method just below?

Reply:
I forgot to delete a method